src/libs/EmojiTrie.ts

```diff
@@ -1,29 +1,46 @@
-import _ from 'underscore';
+import React from 'react';
+import {SvgProps} from 'react-native-svg';
 import emojis, {localeEmojis} from '../../assets/emojis';
 import Trie from './Trie';
 import Timing from './actions/Timing';
 import CONST from '../CONST';
+import {MetaData} from './Trie/TrieNode';
+
+type Emoji = {
+    code: string;
+    header?: boolean;
+    icon?: React.FC<SvgProps>;
+    name?: string;
+    types?: string[];
+};
+
+type LangEmoji = {
+    name?: string;
+    keywords: string[];
+};
+
+type LangEmojis = Record<string, LangEmoji>;
 
 Timing.start(CONST.TIMING.TRIE_INITIALIZATION);
 
 const supportedLanguages = [CONST.LOCALES.DEFAULT, CONST.LOCALES.ES];
 
 /**
  *
- * @param {Trie} trie The Trie object.
- * @param {Array<String>} keywords An array containing the keywords.
- * @param {Object} item An object containing the properties of the emoji.
- * @param {String} name The localized name of the emoji.
- * @param {Boolean} shouldPrependKeyword Prepend the keyword (instead of append) to the suggestions
+ * @param trie The Trie object.
+ * @param keywords An array containing the keywords.
+ * @param item An object containing the properties of the emoji.
+ * @param name The localized name of the emoji.
+ * @param shouldPrependKeyword Prepend the keyword (instead of append) to the suggestions
  */
-function addKeywordsToTrie(trie, keywords, item, name, shouldPrependKeyword = false) {
-    _.forEach(keywords, (keyword) => {
+function addKeywordsToTrie(trie: Trie<MetaData>, keywords: string[], item: Emoji, name: string, shouldPrependKeyword = false): void {
```

Comment:
```suggestion
function addKeywordsToTrie(trie: Trie<MetaData>, keywords: string[], item: Emoji, name: string, shouldPrependKeyword = false) {
```

When the function don't return anything, we don't need to add `void`